tsconfig.check.json

```diff
@@ -0,0 +1,5 @@
+{
+	"extends": "./tsconfig.json",
```

Comment:
ironic :)
```suggestion
  "extends": "./tsconfig.json",
```

Reply:
There we go, that's the downside of me using two-spaced tab indent in GitHub 😃